routes/weather.ts

```diff
@@ -262,31 +229,26 @@ export const getWateringData = async function( req: express.Request, res: expres
 		wateringData = await weatherProvider.getWateringData( coordinates );
 	}
 
-	let scale = -1,	rainDelay = -1;
+	const adjustmentMethodResponse: AdjustmentMethodResponse = await adjustmentMethod.calculateWateringScale(
+		adjustmentOptions, wateringData, coordinates, weatherProvider
+	);
 
-	if ( adjustmentMethod === ADJUSTMENT_METHOD.ZIMMERMAN ) {
-		scale = calculateZimmermanWateringScale( adjustmentOptions, wateringData );
+	// If an error that should be sent to the user occurred, send it to them.
+	if ( adjustmentMethodResponse.errorMessage ) {
```

Comment:
Shouldn't this be an exception from L332 rather than a message in a response. Would be more consistent with the error handling above.

Reply:
To be discussed in #54